pep-0705.rst

```diff
@@ -0,0 +1,259 @@
+PEP: 705
+Title: TypedMapping: Type Hints for Mappings with a Fixed Set of Keys
+Author: Alice Purcell <alicederyn@gmail.com>
+Sponsor: Pablo Galindo <pablogsal@gmail.com>
+Discussions-To: pending
```

Comment:
Didn't realise this would block linting so much. Removing until we have a thread to link to.
```suggestion
```

Reply:
See https://github.com/alicederyn/peps/pull/4#discussion_r1102348723 ; it makes that one check go red as expected, but doesn't interfere with any others.